screens/Home/MyVcsTab.tsx

```diff
@@ -152,6 +154,13 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = (props) => {
         error={storeErrorTranslationPath}
         onDismiss={controller.DISMISS}
       />
+      <MessageOverlay
+        isVisible={controller.isBindingError}
+        title={controller.walletBindingError}
```

Comment:
(opinion) this might preferably be put in the description with a consistent title